doc-content/enterprise-only/openshift/operator-modify-proc.adoc

```diff
@@ -10,7 +10,7 @@ To modify the environment, you must modify the YAML description of the environme
 . In the OpenShift Web console navigation panel, select *Catalog -> Installed operators* or *Operators -> Installed operators*.
 . Find the `Business Automation` operator line in the table and click `KieApp` in the line. Information about the environments that you deployed using this operator is displayed.
 . Click the name of a deployed environment. 
-. Select the *YAML* tab. A YAML source is displayed. 
+. Select the *YAML* tab. A YAML source is displayed. In this YAML source, you can edit the content under `spec:` to change the configuration of the environment.
```

Comment:
We can rewrite this(Just a suggestion)?
. Select the *YAML* tab.
+
A YAML source is displayed. In this YAML source, you can edit the content under `spec:` to change the configuration of the environment.

Seems like the procedure is too long.